example/gardener-local/kind-ha/cluster-local.yaml

```
@@ -0,0 +1,135 @@
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+nodes:
+- role: control-plane
+  topology.kubernetes.io/zone: "0"
+  image: kindest/node:v1.24.0
+  extraPortMappings:
+  # istio-ingressgateway
+  - containerPort: 30443
+    hostPort: 443
+    listenAddress: 127.0.0.1
+  # etcd (for gardener-apiserver)
+  - containerPort: 32379
+    hostPort: 32379
+  # ingress-nginx (Seed)
+  - containerPort: 30448
+    hostPort: 8448
+  # registry for locally built images
+  - containerPort: 5001
+    hostPort: 5001
+  extraMounts:
+  - hostPath: example/gardener-local/controlplane
+    containerPath: /etc/gardener/controlplane
+    readOnly: true
+  - hostPath: dev/local-backupbuckets
+    containerPath: /etc/gardener/local-backupbuckets
+  - hostPath: dev/local-registry
+    containerPath: /etc/gardener/local-registry
+  kubeadmConfigPatches:
+  - |
+    kind: ClusterConfiguration
+    apiServer:
+      extraArgs:
+        authorization-mode: RBAC,Node,Webhook
+        authorization-webhook-config-file: /etc/gardener/controlplane/auth-webhook-kubeconfig-local.yaml
+        authorization-webhook-cache-authorized-ttl: "0"
+        authorization-webhook-cache-unauthorized-ttl: "0"
+      extraVolumes:
+      - name: gardener
+        hostPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-local.yaml
+        mountPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-local.yaml
+        readOnly: true
+        pathType: File
+  - |
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    kind: KubeletConfiguration
+    maxPods: 500
+    serializeImagePulls: false
+    registryPullQPS: 10
+    registryBurst: 20
+
+- role: worker
+  labels:
+    topology.kubernetes.io/zone: "1"
+  image: kindest/node:v1.24.0
+  extraMounts:
+  - hostPath: example/gardener-local/controlplane
+    containerPath: /etc/gardener/controlplane
+    readOnly: true
+  - hostPath: dev/local-backupbuckets
+    containerPath: /etc/gardener/local-backupbuckets
+  kubeadmConfigPatches:
+  - |
+    kind: ClusterConfiguration
+    apiServer:
+      extraArgs:
+        authorization-mode: RBAC,Node,Webhook
+        authorization-webhook-config-file: /etc/gardener/controlplane/auth-webhook-kubeconfig-local.yaml
+        authorization-webhook-cache-authorized-ttl: "0"
+        authorization-webhook-cache-unauthorized-ttl: "0"
+      extraVolumes:
+      - name: gardener
+        hostPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-local.yaml
+        mountPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-local.yaml
+        readOnly: true
+        pathType: File
+  - |
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    kind: KubeletConfiguration
+    maxPods: 500
+    serializeImagePulls: false
+    registryPullQPS: 10
+    registryBurst: 20
```

This way, we don't repeat ourselves:

```suggestion
  kubeadmConfigPatches: *kubeadmConfigPatches
```

---

Fixed.